WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
```

Comment:
Aha, thank you for voicing that! I spent more than a few minutes thinking "hey, this check kinda doesn't make sense to here since it's the _user_, not the Blog that has the capability or not". And started pondering whether this method should be called "hasUserCapability" or something or even extract the method to some other class! At the end, I opted for keeping this simple for now and only proposed renaming it to a more "asking a question" pattern. In other words, I'm open to suggestions here even though this is definitely not a blocker for the PR. Cheers!